scripts/tests/ota_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+PASSCODE=${1:-20202021}
+DISCRIMINATOR=${2:-42}
+UDP_PORT=${3:-5560}
+
+pkill chip-ota-provider-app
+pkill chip-ota-requestor-app
+
+scripts/examples/gn_build_example.sh examples/chip-tool out/
+
+touch my-firmware.bin
+
+./src/app/ota_image_tool.py create -v 0xDEAD -p 0xBEEF -vn 1 -vs "1.0" -da sha256 my-firmware.bin my-firmware.ota
+
+if [ ! -f "my-firmware.ota" ]; then
+    exit 1
+fi
+
+./out/ota_provider_debug/chip-ota-provider-app -f my-firmware.ota | tee /tmp/ota/provider-log.txt &
+provider_pid=$!
+
+echo "Commissioning Provider"
+
+./out/chip-tool pairing onnetwork 1 "$PASSCODE" | tee /tmp/ota/chip-tool-commission-provider.txt
+if grep "Device commissioning completed with success" /tmp/ota/chip-tool-commission-provider.txt; then
+    echo Provider Commissioned
+else
+    echo Provider not commissioned properly
+fi
+
+# TODO: This should be removed once chip_kvs is fixed
+rm /tmp/chip_kvs
```

Comment:
FYI, this is no longer necessary and can now be removed. You can now specify where KVS is stored when launch the application. Example given below.